src/CommunityToolkit.Maui.Core/Platform/Android/MCTPopup.android.cs

```diff
@@ -0,0 +1,92 @@
+﻿using System;
+using Android.App;
+using Android.Content;
+using CommunityToolkit.Maui.Core;
+using Microsoft.Maui.Platform;
+using AView = Android.Views.View;
+
+namespace CommunityToolkit.Core.Platform;
+
+/// <summary>
+/// The navite implementation of Popup control.
+/// </summary>
+public class MCTPopup : Dialog, IDialogInterfaceOnCancelListener
```

Comment:
This will be solved when we define how to name those kinds of controls. I'll resolve others since they will be solved at once and to reduce noise on this PR

Reply:
Hey Pedro! I updated the name from `MCTPopup` to `MauiPopup`.

After taking a look at the .NET MAUI repo, this aligns with their naming convention, e.g. `public class MauiLabel : UILabel`